client/src/api/index.ts

```diff
@@ -0,0 +1,8 @@
+import { operationsByTag } from './components';
+import type * as Responses from './responses';
+import type * as Schemas from './schemas';
+
+export * from './client';
+export * from './components';
+export { operationsByTag as Operations };
+export type { Responses, Schemas };
```

Comment:
Official exports for apps:

1. ``XataApiClient`` constructor based client
2. All fetchers exposed as tree-shakeable functions
3. ``Operations`` dictionary by tags of the fetchers
4. ``Responses``and ``Schemas`` types

We might not want to promote all of them in the docs, maybe only the constructor based implementation, but I like to provide as much tools any experienced developer might want to use.